orttraining/orttraining/python/training/onnxblock/optim/optim.py

```diff
@@ -11,13 +13,19 @@
 
 # TODO: Find a better place for these constants
 _PRODUCER_NAME = "onnxblock offline tooling"
-_OPSET_IMPORTS = [onnx.helper.make_opsetid("com.microsoft", 1), onnx.helper.make_opsetid("", 14)]
+_OPSET_IMPORTS = (onnx.helper.make_opsetid("com.microsoft", 1), onnx.helper.make_opsetid("", 14))
 
 
 class AdamWOptimizer(building_blocks.Block):
     """Adds an AdamWOptimizer node to the onnx model."""
 
-    def __init__(self, bias_correction=True, betas=(0.9, 0.999), eps=1e-6, weight_decay=0.0):
+    def __init__(
+        self,
+        bias_correction: bool = True,
+        betas: tuple[int, int] = (0.9, 0.999),
```

Comment:
tuple[int, int] may fail for python < 3.9. I would do

```python
from typing import Tuple
```

and annotate it as `Tuple[int, int]`, until our minimal supported python version becomes 3.9